src/style.css

```diff
@@ -0,0 +1,118 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin-top: 0;
+}
+
+.form {
+  width: 100%;
+}
+
+.search {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  height: 70px;
+  box-sizing: border-box;
+  margin: 20px 0 0;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.image {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 25px;
+  top: 24px;
+}
+
+.input {
+  position: absolute;
+  left: 61.2px;
+  top: 22.4px;
+  right: 18px;
```

Comment:
input should be full height and full width
to leave some empty space for the image use left padding
<img width="600" alt="image" src="https://user-images.githubusercontent.com/66749919/200109069-68b3363f-377b-4a2d-9499-addad6ebc8de.png">
and don't forget to remove default outline